src/ORM/FieldType/DBDate.php

```diff
@@ -217,7 +217,23 @@ public function getISOFormat()
     }
 
     /**
-     * Return the date using a particular formatting string.
+     * @return string
+     */
+    public function getDayOfMonthOrdinal()
+    {
+        $number = $this->Format('d');
+        if ($number) {
+            $formatter = NumberFormatter::create(i18n::get_locale(), NumberFormatter::ORDINAL);
+            $ordinal = str_replace($number, '', $formatter->format($number)); // Remove number, leaving just ordinal
+            return "'{$ordinal}'";
```

Comment:
Why is the ordinal single quoted?

Reply:
Oh I see, it's to escape for text.

http://userguide.icu-project.org/formatparse/datetime

The problem is that if you use this function by itself, it'll be quoted incorrectly.